src/index.html

```diff
@@ -17,11 +17,389 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
```

Comment:
There are duplicate links to the Google Fonts API. Consider removing one of them to avoid unnecessary requests.